samples/caf_sensor_manager/prj.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_CAF=y
 CONFIG_CAF_INIT_LOG_SENSOR_EVENTS=n
 
 CONFIG_CAF_SENSOR_MANAGER=y
+CONFIG_CAF_SENSOR_MANAGER_THREAD_STACK_SIZE=512
```

Comment:
Yes it is set to 2048. I wouldn't like to decrease default option so much. In case of our sample we have only one simulated sample so 512 B is enough but in case of usage with more sensors it could be too little.